src/thor/multimodal.cc

```diff
@@ -526,6 +526,9 @@ bool MultiModalPathAlgorithm::ExpandForward(GraphReader& graphreader,
 
   // Handle transitions - expand from the end node each transition
   if (!from_transition && nodeinfo->transition_count() > 0) {
+    if (tile == nullptr) {
```

Comment:
this is also not needed, the top of the function checks this condition already to make sure tile is not null